### PR TITLE
docs: add pre-commit todo in the development guide

### LINKED ISSUE #906: docs: add pre-commit todo in the development guide
i.e. `update-license update-mdlint update-gofmt` in Makefile.

### PR DESCRIPTION
**Pull request by jwrookie**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

Resolves #906

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: docs/en/latest/development.md [hunk 1]
```diff
@@ -103,3 +103,18 @@ Something you need to pay attention to:
...
+
+## Pre-commit todo
+
+When everything is ready, before submitting the code, please make sure that the license, code style, and document format are consistent with the project specification.
+
+We provide commands to implement it, just run the following commands:
+```shell
+make update-license
+make update-mdlint
+make update-gofmt
```
Comment (tao12345666333) [inline docs/en/latest/development.md:115]: We also need to add `make update-codegen`

**Review by tao12345666333 — COMMENTED** [review]
Thanks

File: docs/en/latest/development.md (hunk 1 above)
Comment (jwrookie) [reply docs/en/latest/development.md:115]: got it

**Review by tao12345666333 — APPROVED** [review]
Thanks

**Review by starsz — APPROVED** [review]

**Pull request merged**